packages/react-components/react-persona/src/stories/Persona/PersonaDefault.stories.tsx

```diff
@@ -1,4 +1,45 @@
 import * as React from 'react';
 import { Persona, PersonaProps } from '@fluentui/react-persona';
+import { makeStyles } from '@fluentui/react-components';
 
-export const Default = (props: Partial<PersonaProps>) => <Persona {...props} />;
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(3, 1fr)',
+  },
+});
+
+export const Default = (props: Partial<PersonaProps>) => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <Persona
+        name="Kevin Sturgis"
+        primaryText="PrimaryText"
+        secondaryText="Secondary text"
+        tertiaryText="Tertiary text"
+        quaternaryText="Quaternary text"
+        {...props}
+      />
+      <Persona
+        name="Kevin Sturgis"
+        presence={{ status: 'out-of-office' }}
+        primaryText="PrimaryText"
+        secondaryText="Secondary text"
+        tertiaryText="Tertiary text"
+        quaternaryText="Quaternary text"
+        {...props}
+      />
+      <Persona
+        presenceOnly
+        presence={{ status: 'out-of-office' }}
+        primaryText="PrimaryText"
+        secondaryText="Secondary text"
+        tertiaryText="Tertiary text"
+        quaternaryText="Quaternary text"
+        {...props}
+      />
+    </div>
+  );
```

Comment:
The Default story should be a single Persona that showcases its common features (e.g. maybe an avatar+presence, with primary and secondary text). The other stories can show examples of the other variants like `presenceOnly`, or with more lines of text, etc.